owncloudApp/src/main/res/values/strings.xml

```diff
@@ -475,6 +475,7 @@
     <string name="network_error_socket_timeout_exception">The server took too long to respond</string>
     <string name="network_error_connect_timeout_exception">The server took too long to connect</string>
     <string name="network_host_not_available">Server could not be reached</string>
+    <string name="network_error_message">An error occurred in the network</string>
```

Comment:
We already have a similar message some lines above `network_error_socket_exception`. Would it be enough for this error?

Reply:
IMO, `network_error_socket_exception` ("An error occurred while connecting with the server.") suggests the server is or could be involved. In this new error message, we only blame the network since we are not able to reach the server, it's like a previous step